spec/calendar.mdwn

```diff
@@ -12,22 +12,44 @@ A **Calendar** object has the following properties:
   Any valid CSS colour value. The colour to be used when displaying events associated with the calendar. The colour SHOULD have sufficient contrast to be used as text on a white background.
 - **isVisible**: `Boolean`
   Should the calendar's events be displayed to the user at the moment?
+- **parentId**: `String|null`
```

Comment:
I don't think we want to make calendars hierarchical; this will hinder compatibility with existing systems, since CalDAV (as far as I'm aware) and all existing systems I've seen don't have child calendars.